tests/acceptance/30_generic_methods/file_ensure_lines_present_test_already_present.cf

```diff
@@ -26,7 +26,7 @@ bundle agent init
     "tmp"         string => getenv("TEMP", 1024);
     "file"        string => "${tmp}/test";
     "file_canon"  string => canonify("${file}");
-    "line_to_add" string => "This is a test line!";
+    "line_to_add" string => "This is a test line.";
```

Comment:
Using "!" caused havoc because of the shell interpreting it in the test results.